__tests__/storage/common_store.test.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LevelDBStore, MemoryStore } from "../../src/storage";
+import { HDPrivateKey } from "bitcore-lib";
+import { TOKEN_AUTHORITY_MASK, TOKEN_MINT_MASK } from "../../src/constants";
+
+const DATA_DIR = './testdata.leveldb';
+
+describe("locked utxo methods", () => {
```

Comment:
I will start to port the other store tests to this file with a similar approach in another PR.

Basically we should have a common tests that can run with any store implementation to make sure all `IStore` implementations behave the same way.
Ideally we should do the same with `IStorage` implementations and check that any combination of `IStorage` and `IStore` will work as intended.

Reply:
I agree. The test codes got too similar as well, it will be good to have them unified.

Reply:
https://github.com/HathorNetwork/hathor-wallet-lib/issues/488